src/Jackett.Common/Indexers/HDSpace.cs

```diff
@@ -102,90 +96,93 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
+                    //var qDownload = row.ChildElements.ElementAt(3).FirstElementChild.Cq();
+                    var qDownload = row.Children[3].FirstElementChild;
+                    release.Link = new Uri(SiteLink + qDownload.GetAttribute("href"));
 
                     //"July 11, 2015, 13:34:09", "Today at 20:04:23"
-                    var dateStr = row.ChildElements.ElementAt(4).Cq().Text().Trim();
-                    if (dateStr.StartsWith("Today"))
-                        release.PublishDate = DateTime.Today + TimeSpan.ParseExact(dateStr.Replace("Today at ", ""), "hh\\:mm\\:ss", CultureInfo.InvariantCulture);
-                    else if (dateStr.StartsWith("Yesterday"))
-                        release.PublishDate = DateTime.Today - TimeSpan.FromDays(1) + TimeSpan.ParseExact(dateStr.Replace("Yesterday at ", ""), "hh\\:mm\\:ss", CultureInfo.InvariantCulture);
-                    else
-                        release.PublishDate = DateTime.SpecifyKind(DateTime.ParseExact(dateStr, "MMMM dd, yyyy, HH:mm:ss", CultureInfo.InvariantCulture), DateTimeKind.Local);
-
-                    var sizeStr = row.ChildElements.ElementAt(5).Cq().Text();
+                    //var dateStr = row.ChildElements.ElementAt(4).Cq().Text().Trim();
+                    var dateStr = row.Children[4].Text().Trim();
+                    //if (dateStr.StartsWith("Today"))
+                    //    release.PublishDate = DateTime.Today + TimeSpan.ParseExact(dateStr.Replace("Today at ", ""), "hh\\:mm\\:ss", CultureInfo.InvariantCulture);
+                    //else if (dateStr.StartsWith("Yesterday"))
+                    //    release.PublishDate = DateTime.Today - TimeSpan.FromDays(1) + TimeSpan.ParseExact(dateStr.Replace("Yesterday at ", ""), "hh\\:mm\\:ss", CultureInfo.InvariantCulture);
+                    //else
+                    //    release.PublishDate = DateTime.SpecifyKind(DateTime.ParseExact(dateStr, "MMMM dd, yyyy, HH:mm:ss", CultureInfo.InvariantCulture), DateTimeKind.Local);
+                    release.PublishDate = DateTimeUtil.FromUnknown(dateStr.Replace("at", string.Empty));
+                    //var sizeStr = row.ChildElements.ElementAt(5).Cq().Text();
+                    var sizeStr = row.Children[5].Text();
                     release.Size = ReleaseInfo.GetBytes(sizeStr);
-
-                    release.Seeders = ParseUtil.CoerceInt(row.ChildElements.ElementAt(7).Cq().Text());
-                    release.Peers = ParseUtil.CoerceInt(row.ChildElements.ElementAt(8).Cq().Text()) + release.Seeders;
-
-                    var grabs = qRow.Find("td:nth-child(10)").Text();
+                    release.Seeders = ParseUtil.CoerceInt(row.Children[7].Text());
+                    release.Peers = ParseUtil.CoerceInt(row.Children[8].Text()) + release.Seeders;
+                    //var grabs = qRow.Find("td:nth-child(10)").Text();
+                    var grabs = row.QuerySelector("td:nth-child(10)").Text();
```

Comment:
Are you sure `row.QuerySelector("td:nth-child(10)")` works as expected? I remember having problems with something similar...